src/styles/botui.scss

```diff
@@ -52,9 +52,11 @@
   text-decoration: underline;
 }
 
-.botui-actions-buttons-button {
-  & + .botui-actions-buttons-button {
-    margin-left: 10px;
+button.botui-actions-buttons-button {
```

Comment:
I had to add `button.` to the selector in order to override the `margin: 0` in `normal.scss`.